### ISSUE TITLE
Always asking for root pw?

### ISSUE THREAD

**Issue by sricks** [muennich/physlock#89]
Hello! When I run `physlock` from my X session, it asks me for the root password in order to unlock `root: password:`. Is this normal?

I use lxdm to as a greeter, and just run `physlock` or `physlock -m` from a xfce4-terminal

If I run physlock from a non-x shell on a different tty, it does lock and unlock as my user.

I am on arch linux, have installed physlock -v 13 from the AUR. I have the following PAM file that the packagers include in /etc/pam.d/physlock

```
#%PAM-1.0
auth		required	pam_unix.so
account		required	pam_unix.so
password	required	pam_unix.so
session		required	pam_unix.so
```

I also tried replacing it with the default pam file you provided: https://github.com/muennich/physlock/blob/master/physlock.pam

I hope this is the right place to report this, thanks!

**Comment by muennich** [comment]
Can you please post the output of `physlock -v`? Release v13 contains a bug that causes physlock not to query systemd-logind. This bug is already fixed in master by commit 2eb17b1111d2839ba24a0ac764a50c29a03c8355.

**Comment by sricks** [comment]
Thanks, it was release v13. I just built from git and the issue is resolved. I'll notify the aur maintainer.

By the way, I found it hard to understand which was the latest version of physlock vs which one I had installed. I assumed it was already up to date, or I would have tried making. I think it could be a nice idea to include the latest v number or bug fix history somewhere in the readme if you have time.

Thank you for this project :)
